### PR TITLE
Created Contacts Component/31

### PR DESCRIPTION
**Pull request by Anushaca1995**
Created .jsx and .scss files for contacts component and passed the appropriate props.

https://trello.com/c/zs8RfeFL/31-create-a-contact-component

### CODE REVIEW

**Review by Martinelli-89 — CHANGES_REQUESTED** [review]
- Missing background color
- Use rem instead of font sizes for font size
- Use rem for image sizes as well
- Delete button and bank imaged same size 

Thanks!

File: src/components/Contacts/Contacts.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    color: $color-text;
+    font-weight: 400;
+    font-size: 0.8rem;
+    line-height: 1.25rem;
+    width: 65rem;
+    height: 3.63rem;
+    font-family: $font-heading;
+    background: $color-background-light;
+    padding: 0.63rem;
+    &__Image{
```
Comment (kirsty-nology) [inline src/components/Contacts/Contacts.scss:19]: lower case i for __image

File: src/components/Contacts/Contacts.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    flex-direction: row;
+    justify-content: center;
+    text-align: center;
+    margin: auto;
+    color: $color-text;
+    font-weight: 400;
+    font-size: 0.8rem;
+    line-height: 1.25rem;
+    width: 65rem;
+    height: 3.63rem;
```
Comment (kirsty-nology) [inline src/components/Contacts/Contacts.scss:15]: this too specific, do either 3.5 or 4

File: src/components/Contacts/Contacts.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import "./Contacts.scss";
+
+const Contacts = ({bankImage, userName, sortCode, accNo, bankName, iban, deleteIcon}) =>{
+
+    return(
+        <div className="contacts">
+            <img className="contacts__Image contacts__Image--bankImage" src={bankImage}></img>
```
Comment (kirsty-nology) [inline src/components/Contacts/Contacts.jsx:7]: lowercase i for `image`

**Review by Zain-Ashfaq — APPROVED** [review]
all works

**Review by Martinelli-89 — APPROVED** [review]

**Review by kirsty-nology — APPROVED** [review]

**Pull request merged**